Hanabi/Player.py

```diff
@@ -39,6 +46,16 @@ def receive_clue(self, clue):
     print(test.hand)
     print('')
 
+    test.receive_clue('Green')
```

Comment:
Keeping them on Card, Arjun mentioned moving them but then he and I agreed it'd be bad.